src/libutil/util.hh

```diff
@@ -292,6 +292,11 @@ std::string drainFD(int fd, bool block = true, const size_t reserveSize=0);
 
 void drainFD(int fd, Sink & sink, bool block = true);
 
+/**
+ * Returns the amount of threads available if possible, otherwise returns 1.
```

Comment:
```suggestion
 * Returns the amount of threads available for simultaneous execution if possible, otherwise returns 1.
```

OS may have a limit on total number of (mostly) inactive threads. This removes that ambiguity.

Reply:
Fixed 👍